chart/compass/templates/tenant-fetcher-job.yaml

```diff
@@ -1,10 +1,40 @@
+{{ $outerScope := . }}
 {{- range $tenantFetcherName, $config := .Values.global.tenantFetchers -}}
 {{if eq $config.enabled true}}
+{{ $configmapName := printf "%s-tenant-fetcher-%s-config" $.Chart.Name $tenantFetcherName | quote }}
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ $configmapName }}
+  namespace: {{ $config.configMapNamespace }}
+  labels:
+    app: {{ $.Chart.Name }}
+    release: {{ $.Release.Name }}
+    helm.sh/chart: {{ $.Chart.Name }}-{{ $.Chart.Version | replace "+" "_" }}
+    app.kubernetes.io/name: {{ template "name" $outerScope }}
+    app.kubernetes.io/managed-by: {{ $.Release.Service }}
+    app.kubernetes.io/instance: {{ $.Release.Name }}
+{{- $configmap := (lookup "v1" "ConfigMap" $config.configMapNamespace $configmapName) }}
+{{- if empty $configmap }}
+data:
+  lastConsumedTenantTimestamp: "1"
```

Comment:
Haha, yes, "1" stands for exactly that, the beginning of time ⌛

Reply:
At an earlier stage, we also thought about this idea and we even implemented it. But after a discussion with parts of the team, we decided to skip this because 1. It looks not very pretty in the code to parameterize properties of a yaml file (because that's one of the places where this key is used) and 2. when someone changes this property, he can easily assume that this key is surely used somewhere else and he would make additional corresponding where it's necessary. Do you agree? If you think that it's good to parametrize, we'll make the changes 👍🏼

Reply:
Yeaah, sure, I think it's fine 👍